src/connector/src/sink/file_sink/s3.rs

```diff
@@ -169,58 +169,3 @@ impl OpendalSinkBackend for S3Sink {
         }
     }
 }
-
-// Wrapper for expanding in the sink config type.
-#[serde_as]
-#[derive(Deserialize, Debug, Clone, WithOptions)]
-pub struct SnowflakeConfig {
-    #[serde(flatten)]
-    pub inner: S3Config,
-}
-
-#[derive(Debug, Clone, Copy, PartialEq, Eq)]
-pub struct SnowflakeSink;
```

Comment:
As we discussed offline, let's keep the original snowflake sink for backward compatibility and name the new snowflake sink `snowflake_v2`.